pkgs/development/python-modules/walrus/default.nix

```diff
@@ -1,4 +1,5 @@
 { lib
+, stdenv
```

Comment:
Why this change?

Reply:
It was a leftover from an in between state.